adapters/handlers/graphql/local/get/class_builder.go

```diff
@@ -101,9 +101,21 @@ func (b *classBuilder) classObject(class *models.Class) *graphql.Object {
 			for _, property := range class.Properties {
 				propertyType, err := b.schema.FindPropertyDataType(property.DataType)
 				if err != nil {
-					// We can't return an error in this FieldsThunk function, so we need to panic
-					panic(fmt.Sprintf("buildGetClass: wrong propertyType for %s.%s; %s",
-						class.Class, property.Name, err.Error()))
+					if err.Error() == schema.ErrRefToNonexistentClass {
```

Comment:
Comparing error values with strings is not considered a good practice because it is very error prone (will break when string changes or when the error is wrapped inside a another one)
Try to define ErrRefToNonexistentClass as an error and use errors.Is() to check for specific error.

Reply:
Done